src/components/Loader/index.tsx

```diff
@@ -1 +1 @@
-export * from './Loader';
+export { Loader } from './Loader';
```

Comment:
Everything looks good in this file. The export was modified according to the rules. Good job! 😊